ksqldb-serde/src/main/java/io/confluent/ksql/serde/connect/ConnectDataTranslator.java

```diff
@@ -41,8 +43,15 @@
 
   private final Schema schema;
 
+  private final Optional<Schema> phyiscalSchema;
+
   public ConnectDataTranslator(final Schema schema) {
+    this(schema, null);
+  }
+
+  public ConnectDataTranslator(final Schema schema, final Schema physicalSchema) {
```

Comment:
Seems that only one schema is used. Why do we need to keep the two schemas in this class? I don't see other code that requires the original even if the physical is present. For instance, `ProtobufSerdeFactory` creates a different `ConnectDataTranslator` depending of the type of schema. Can' we just keep one schema? or if the reason is that you want to know whether this schema is physical or not, then should I different class that extends from `ConnectDataTransaltor` be sufficient?

Reply:
Good point. Let me refactor and create a separate class extending `ConnectDataTranslator`